ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -851,20 +1001,28 @@ export const RuntimePanel = fp.flow(
     computeType: initialCompute,
     machine: initialMasterMachine,
     diskSize: diskSize,
-    dataprocConfig: dataprocConfig
+    dataprocConfig: dataprocConfig,
+    pdSize: pdSize
   };
 
   const newRuntimeConfig = {
     computeType: selectedCompute,
     machine: selectedMachine,
     diskSize: selectedDiskSize,
-    dataprocConfig: selectedDataprocConfig
+    dataprocConfig: selectedDataprocConfig,
+    pdSize: selectedPdSize
   };
 
   const runtimeDiffs = getRuntimeConfigDiffs(initialRuntimeConfig, newRuntimeConfig);
   const runtimeChanged = runtimeExists && runtimeDiffs.length > 0;
+  const pdSizeReduced = pdExists && selectedPdSize < pdSize;
+  const pdSizeNotReduced = pdExists && selectedPdSize >= pdSize;
```

Comment:
Please reduce/simplify where possible - do you need both of these variables, or can yo ujust `!` one of them? Some of the predicates below are getting pretty long and difficult to read.

Reply:
`!pdSizeReduced` would include PD not exist OR pdSize increase. If omitting,  then then line where pdSizeNotReduced would become (`pdExists && !pdSizeReduced`) which also seems fine.

Reply:
agree, I edit it with the second one